src/const.js

```diff
@@ -1,3 +1,9 @@
+import {
```

Comment:
The constants file shouldn't import anything else; I worry about circular dependencies (even if they work, which I forget if they do or not), it makes things really hard to follow.

Constants for us are largely named variables to avoid [magic constants](https://en.wikipedia.org/wiki/Magic_number_%28programming%29#Unnamed_numerical_constants).

Can this be refactored to avoid the `import`?